modin/pandas/__init__.py

```diff
@@ -337,6 +351,13 @@ def init_remote_ray(partition):
     "datetime",
     "NamedAgg",
     "api",
+    "read_xml",
 ]
 
+if PandasCompatVersion.CURRENT != PandasCompatVersion.PY36:
+    from modin._compat.pandas_api.namespace import Flags, Float32Dtype, Float64Dtype
+
+    __all__.extend(["Flags", "Float32Dtype", "Float64Dtype"])
+del PandasCompatVersion
```

Comment:
Probably there already was a related question. Could you please recall why we delete it?

Reply:
Sure. There's a test which checks that there's nothing extra exposed from `modin.pandas` namespace compared to `pandas` (with a few exceptions). We don't need a user to use this class, and there's no point in making it available from `modin.pandas`, so I'm removing it here instead of expanding the exclusion list in the test.

Reply:
I see, thanks!